interfaces/PA-App/src/app/personal-components/phone-number/phone-number.component.ts

```diff
@@ -51,6 +51,7 @@ export class PhoneNumberComponent extends PersonalComponent {
   async initNew() {
     this.languageCode = this.translate.currentLang;
     this.did = await this.paData.retrieve(this.paData.type.did);
+    this.prefillPhoneNumber();
```

Comment:
I think, for now, we'll have enough with:
```suggestion
    this.phoneNumber = await this.paData.retrieve(this.paData.type.phoneNumber);
```